windows/make.vbs

```diff
@@ -1,2 +1,99 @@
+
+'Build script for KOLIBRI INSTALLER WINDOWS
+'
+'Steps
+'
+'1. Set Kolibri Version
+'2. Download python installer
+'3. Build the installation package
+'
+'Python Variable
+'PYTHON_VERSION == must set a specific verion (e.g. "2.7", "3.4.3" and etc.)
+'PYTHON_32BIT_LINK & PYTHON_64BIT_LINK == URL of the python .exe 
+'PYTHON_DIR == Directory when python must save
+'
+'REF: VB Scripting References
+'https://www.codeproject.com/tips/506439/downloading-files-with-vbscript
+'http://www.csidata.com/custserv/onlinehelp/vbsdocs/vbs14.htm
+'https://stackoverflow.com/questions/20626863/can-you-get-the-pc-user-name-in-vbs
+'https://blogs.technet.microsoft.com/heyscriptingguy/2005/02/08/how-can-i-find-and-replace-text-in-a-text-file/
+
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Function Downloads(strLink,path)
+'strLink == Get file name from URL.
+'path == for directory file
+strSaveName = Mid(strLink, InStrRev(strLink,"/") + 1, Len(strLink))
+strSaveTo = path & strSaveName
+' Create an HTTP object
+Set objHTTP = CreateObject("MSXML2.XMLHTTP")
+' Download the specified URL
+objHTTP.open "GET", strLink, False
+objHTTP.send
+ 
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+If objFSO.FileExists(strSaveTo) Then
+  objFSO.DeleteFile(strSaveTo)
+End If
+
+If objHTTP.Status = 200 Then
+  Dim objStream
+  Set objStream = CreateObject("ADODB.Stream")
+  With objStream
+    .Type = 1 'adTypeBinary
+    .Open
+    .Write objHTTP.responseBody
+    .SaveToFile strSaveTo
+    .Close
+  End With
+  set objStream = Nothing
+End If
+
+end Function

```

Comment:
Function body needs indentation.

I'm not a Windows expert, but is it guaranteed that this function fails reliably if the download is broken?

I worry in case we bundle a broken installer...

@mrpau-eduard @cpauya - do you have experience with this?

Reply:
Good points @benjaoming - I haven't looked at VBScript for a while so pardon me. 

Let's put some error trapping codes @mrpau-julius - to make sure we fail reliably in case of broken downloads.  Here's a reference for you [VBScript — Using error handling](https://stackoverflow.com/a/157785/845481).

And yes, please make your code beautiful to read. ;p

Reply:
thank you @cpauya 😃, I will study that link. 😃